### ISSUE TITLE
Add a "settings" tab

### ISSUE THREAD

**Issue by gfaugere** [SThor/incremental#10]
Need to consolidate the "Skip prolog" and "Reset state" buttons in it

**Comment by SThor** [comment]
Good idea :)  However I would prefer a small modal window tucked in a corner, and with its visibility toggable in some way, via a simple quiet button or a keypress.
